provider/cluster/kube/statefulset_builder.go

```diff
@@ -0,0 +1,72 @@
+package kube
+
+import (
+	"github.com/tendermint/tendermint/libs/log"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/ovrclk/akash/manifest"
+	mtypes "github.com/ovrclk/akash/x/market/types"
+)
+
+type statefulSetBuilder struct {
+	workloadBuilder
+}
+
+func newStatefulSetBuilder(log log.Logger, settings Settings, lid mtypes.LeaseID, group *manifest.Group, service *manifest.Service) *statefulSetBuilder {
+	return &statefulSetBuilder{
+		workloadBuilder: newWorkloadBuilder(log, settings, lid, group, service),
+	}
+}
+
+func (b *statefulSetBuilder) create() (*appsv1.StatefulSet, error) { // nolint:golint,unparam
+	replicas := int32(b.service.Count)
```

Comment:
Do we need to check this for being 0 or less? It seems weird kubernetes even allows an `int` type instead of `uint`